pkg/models/incorrect_model_status_meta.go

```diff
@@ -0,0 +1,87 @@
+package models
+
+import (
+	"database/sql/driver"
+	"encoding/json"
+	"errors"
+
+	"github.com/google/uuid"
+)
+
+// IncorrectModelStatusActivityMeta represents the notification data for an incorrect model status activity
+type IncorrectModelStatusActivityMeta struct {
```

Comment:
I think we might have talked already and came to the idea of a string being stored, but I wonder if it makes more sense to store the raw model status type? I guess the point would be that it could get difficult if a status changed and we no longer have the translation? @patrickseguraoddball, what do you think?

Reply:
I switched this to be the not humanized strings, unfortunately can't really utilize the phaseSuggesstion model in the model package since the models package (generated types) references the model package

Reply:
this is now embedded type for PhaseSuggestion